README.md

```diff
@@ -3,23 +3,41 @@ This is a web interface for [Plagiabot's Copyright RC feed](https://en.wikipedia
 
 [![Build Status](https://travis-ci.org/wikimedia/CopyPatrol.svg?branch=master)](https://travis-ci.org/Niharika29/PlagiabotWeb)
 
-#### To test locally:
-1. Run `composer install` and `composer update` after cloning the repository
-2. Setup your .env file with the following params:
-	* OAUTH_CONSUMER_TOKEN
-	* OAUTH_SECRET_TOKEN
-	* OAUTH_ENDPOINT
-	* OAUTH_REDIR
-	* OAUTH_CALLBACK
-	* DB_DSN_PLAGIABOT
-	* DB_DSN_ENWIKI
-	* DB_USER
-	* DB_PASS
-3. Rewrite your routing locally, if needed
+## To install locally
+1. Clone the repository and run `composer install`.
+2. Edit the `.env` file that was created by composer.
+   1. Get Oauth tokens by registering a new consumer on Meta
+      at [Special:OAuthConsumerRegistration](https://meta.wikimedia.org/wiki/Special:OAuthConsumerRegistration).
+   2. To use Redis caching, also add `REDIS_HOST` and `REDIS_PORT`;
+      without these, a local filesystem cache will be used
+      (in the `cache/` directory, which you may have to create);
+5. Rewrite your routing, if needed.
+   For Lighttpd, use this in your `.lighttpd.conf`:
+   ```
+   url.rewrite-if-not-file += ( "(.*)" => "/copypatrol/index.php/$0" )
+   ```
+   Or for Apache, this (in `.htaccess` at the root of the project):
+   ```
+   DirectorySlash Off
+   RewriteEngine On
+   RewriteCond %{REQUEST_FILENAME} !-f
+   RewriteRule ^public_html(.*)$ public_html/index.php$1 [L]
+   ```
+6. Open up an SSH tunnel to access the databases on Tool Labs (substitute your own username).
+   ```
+   $ ssh -L 4711:enwiki.labsdb:3306 YOU@tools-login.wmflabs.org -N 
+   ```
+7. Set up a local copy of the 'copyright' database (substitute your own usernames):
+   ```
+   $ mysqldump -u LABSYOU -p -P 4711 -h 127.0.0.1 --skip-lock-tables s51306__copyright_p > copyright.sql
+   $ mysql -u LOCALYOU -p -e "CREATE DATABASE copypatrol_dev;"
+   $ mysql -u LOCALYOU -p copypatrol_dev < copyright.sql
```

Comment:
The `s51306__copyright_p` table is what's being continually updated by EranBot, so you'll have a continuous flow of new data, at least until people stop adding copyright violations :) It is true our users go through the backlog pretty quickly, so if you wanted to test something against a lot of unreviewed cases you probably won't be able to. For most dev work I don't think this will be a problem, though